cfg/config.go

```diff
@@ -3,11 +3,97 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+// Load configs from a env file & sets them in environment variables .
+func LoadEnvVar() error {
+	f, err := os.Open(".env")
+
+	if err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	defer func() {
+		err := f.Close()
+		if err != nil {
+			log.Printf("%s", err)
+		}
+	}()
+
+	lines := make([]string, 0)
+	scanner := bufio.NewScanner(f)
+
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	for _, l := range lines {
+		pair := strings.Split(l, "=")
+		os.Setenv(pair[0], pair[1])
+	}
+	return nil
```

Comment:
It seems that you lost empty row under return

Reply:
> It seems that you lost empty row under return

Eagle eye), thank you! Added empty row